SofascoreAcademy/SofascoreAcademy/Views/MainViewController.swift

```diff
@@ -0,0 +1,166 @@
+//
+//  MainViewController.swift
+//  SofascoreAcademy
+//
+//  Created by Akademija on 08.04.2024..
+//
+
+import Foundation
+import UIKit
+import SofaAcademic
+import SnapKit
+
+protocol parentSportSlugPicker {
+    func getPressedTab(request: sport_slug?)
+}
+
+protocol settingsButtonTap {
```

Comment:
Veliko slovo i razmak, ali opcenito naming. Koristis ovaj protocol da bi implementiro delegate pattern tako da bi on trebao sadrzavati to u svom imenu, npr SettingsViewControllerDelegate